src/Notifier.php

```diff
@@ -61,29 +60,14 @@ public function __construct(Client $client, ImmutableConfig $config, LoggerInter
   /**
    * Send a notification to a Meteor server instance.
    *
-   * @param string $event
-   *   The name of the event to notify the instance about.
-   * @param \Drupal\Core\Session\AccountInterface $account
-   *   Optional. The account triggering the event.
-   * @param int $usDelay
-   *   Optional. The delay after which to act on the notification.
+   * @param string $path
+   *   The path reached in meteor.
+   * @param array $query
+   *   Optional query params.
    */
-  public function notify($event, AccountInterface $account = NULL, $usDelay = 0) {
-    $query = [
-      'event' => strval($event),
-    ];
-
-    $userId = $account instanceof AccountInterface ? intval($account->id()) : NULL;
-    if (isset($userId)) {
-      $query['userId'] = $userId;
-    }
-
-    if (isset($usDelay)) {
-      $query['delay'] = abs(intval($usDelay));
-    }
-
+  public function notify($path, $query = []) {
     $host = $this->config->get('meteor_server');
-    $url = $host . '/' . static::PATH;
+    $url = $host . '/' . $path;
```

Comment:
Now that path is a parameter instead of a known constant, it needs to be sanitized, notably for possible excess slashes. Also possibly need to strip any query or fragment which may be present to avoid building inconsistent URLs ( e.g. $path="/foo#bar?baz=quux"  -> merge baz=quux to query and drop excess /)